gemini/sample-apps/photo-discovery/app/lib/config.dart

```diff
@@ -0,0 +1,2 @@
+const geminiModel = 'gemini-1.5-pro-preview-0409';
```

Comment:
```suggestion
const geminiModel = 'gemini-1.5-pro-001';
```
Preview will be deprecated soon

Reply:
Thank you for catching! I meant to update that for this sample.